admin/app/components/target-profiles/update-target-profile.js

```diff
@@ -79,16 +79,17 @@ export default class UpdateTargetProfile extends Component {
 
   async _updateTargetProfile() {
     const model = this.args.model;
-    model.name = this.form.name.trim();
-    model.description = this.form.description ? this.form.description.trim() : null;
-    model.comment = this.form.comment ? this.form.comment.trim() : null;
+    model.name = this.form.name;
+    model.description = this.form.description ? this.form.description : null;
+    model.comment = this.form.comment ? this.form.comment : null;
```

Comment:
**suggestion** : ici comme on met description et comment en nullableText on a plus besoin de la condition :
`model.description = this.form.description;`
`model.comment = this.form.comment;`